common/lib/capa/capa/responsetypes.py

```diff
@@ -2546,7 +2546,7 @@ def setup_response(self):
         super(SymbolicResponse, self).setup_response()  # lint-amnesty, pylint: disable=super-with-arguments
 
     def execute_check_function(self, idset, submission):
-        from symmath import symmath_check
+        from symmath.symmath_check import symmath_check
```

Comment:
Hm, what is making this import change necessary?

Keep in mind that because [symmath has been available in the edx-sandbox (ie, codejail) environment](https://github.com/openedx/edx-platform/blob/5708787527453bbaa11ed90d1d98a35ec3ca1388/requirements/edx-sandbox/py38.txt#L9), we must assume that symmath's interface is used in some instructor-authored grading code that we do not have visibility into. Because of that, we cannot make any breaking symmath API changes.

Reply:
When `symmath` was moved to `openedx-calc`, the following was removed from `symmath/__init__.py`
```
# lint-amnesty, pylint: disable=missing-module-docstring
from .formula import *
from .symmath_check import *
```

This allows the importation directly from `symmath_check`.
This can be re-added quickly to `openedx-calc`.

Reply:
Sounds great.

Reply:
Related: https://github.com/openedx/openedx-calc/pull/41